src/status_im/contexts/wallet/account/share_address/view.cljs

```diff
@@ -34,7 +34,8 @@
   [selected-networks]
   (let [on-save       (fn [chain-ids]
                         (rf/dispatch [:hide-bottom-sheet])
-                        (reset! selected-networks (map utils/id->network chain-ids)))
+                        (reset! selected-networks (map #(if (= % 1) :mainnet (utils/id->network %))
+                                                       chain-ids)))
```

Comment:
I am wondering why did we change the naming of the ethereum network in db to `mainnet`. This caused several instances where ethereum network needs to be handled with a condition, also like [this](https://github.com/status-im/status-mobile/blob/develop/src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs#L50) and [this](https://github.com/status-im/status-mobile/blob/develop/src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs#L148)

cc @J-Son89 @smohamedjavid @briansztamfater

Reply:
I'm not sure, but in the designs it's sometimes mainnet and sometimes Ethereum so that might have affected why. 
Sounds good to choose one 👍

Reply:
I have opened an issue https://github.com/status-im/status-mobile/issues/19481

Reply:
> Additionally, this issue was fixed in the PR https://github.com/status-im/status-mobile/pull/19150. It's waiting for Manual QA 😓 .

Oh okay good to know the issue has already been addressed